src/NUnitFramework/framework/Internal/Commands/OneTimeSetUpCommand.cs

```diff
@@ -65,15 +65,8 @@ public override TestResult Execute(TestExecutionContext context)
             if (_typeInfo != null)
             {
                 // Use pre-constructed fixture if available, otherwise construct it
-                if (!_typeInfo.IsStaticClass)
-                {
-                    context.TestObject = _suite.Fixture ?? _typeInfo.Construct(_arguments);
-                    if (_suite.Fixture == null)
-                    {
-                        _suite.Fixture = context.TestObject;
-                    }
-                    Test.Fixture = _suite.Fixture;
-                }
+                if (!_typeInfo.IsStaticClass && context.TestObject == null)
+                    context.TestObject = _typeInfo.Construct(_arguments);
```

Comment:
Normally, they are created in one time setup. If they implement IDisposable, they are disposed in OneTimeTearDown, so that's where they should go out of scope else we get an error for referencing a disposed object. They should be copied down to child contexts where appropriate. One issue I can imagine is that they should NOT be copied down from SetUpFixtures. Hmmm... have to check how we do that.

The test utilities have to create some tests and run them in a non-normal manner. They sometimes pre-create a fixture and store it in the Fixture property of the test. But they also store it in the context, which ought to be enough. And we can always change our test utilities if necessary.